snooty.toml

```diff
@@ -11,3 +11,5 @@ pgp-version = "{+version+}"
 api = "https://mongodb.github.io/node-mongodb-native/{+version+}"
 mongosh = "``mongosh``"
 driver = "node"
+driver-long = "MongoDB Node.js driver"
+driver-short = "Node.js driver"
```

Comment:
Is this PR only for a single version (no cherry-pick)? If so, I wonder if you might consider separating it out so that you can make this specific change available to all versions.

Reply:
Good point. [I've broke this into a separate PR](https://github.com/mongodb/docs-node/pull/276). Could you review it when you get the chance?